integration-tests/tests/aws/kbv-check-happy.test.ts

```diff
@@ -0,0 +1,67 @@
+import { stackOutputs } from "./resources/cloudformation-helper";
+import { clearItems, populateTable } from "./resources/dynamodb-helper";
+import { executeStepFunction } from "./resources/stepfunction-helper";
+
+describe("HMRC KBV Check ", () => {
+  const stateMachineInput = {
+    sessionId: "12345",
+  };
+  const testUser = {
+    sessionId: "12345",
+    nino: "AA000003D",
+  };
+
+  const testQuestions = [
+    {
+      sessionId: "12345",
+      questionKey: "ita-bankaccount",
+      answered: "false",
+      correlationId: "93dcc67c-fe6d-4bd7-b68f-2bd848e0d572",
+      info: new Map(),
+    },
+    {
+      sessionId: "12345",
+      questionKey: "ita-bankaccount2",
+      answered: "false",
+      correlationId: "93dcc67c-fe6d-4bd7-b68f-2bd848e0d573",
+      info: new Map(),
+    },
+  ];
+
+  let output: Partial<{
+    PersonalIdenityTable: string;
+    QuestionsTable: string;
+    QuestionStateMachineArn: string;
+  }>;
+  beforeEach(async () => {
```

Comment:
We need another scenario to test the user coming into the system for the first time i.e. there are no questions currently in the database. This means that we will need to use the imposter API which is something we are going to be using in the future for the get-ivq-questions state machine. For now, it should be okay to use the HMRC sandbox URL that we currently use.